### DISCUSSION TITLE
Is there a Vanilla JS version of PrimeNG or at least what looks the closest?

### DISCUSSION THREAD

**Discussion by shellwe** [primefaces/community#2956] [PrimeNG]
We have an Angular site and a CMS site and want them to look consistent, so I was wondering what is the closest non-js framework component library that is like PrimeNg?

**Comment by qwadrox** [comment]
@shellwe Hi! I don't think there is as much complete lib as PrimeNG is. I know that they're planning to have a web-component (Lit) library in the Prime component family, but most likely it won't happen very soon. But you might find alternatives that are suitable for you, like: 
https://shoelace.style/
https://open-ui.org/

**Reply by shellwe** [reply]
Thanks for the suggestion! We are taking a look at those now. I hope they do come out with a Lit option soon, but we will likely already have another solution before they do.

Thanks again!
